app/scripts/views/sign_up.js

```diff
@@ -177,6 +177,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this.getUserAgent().hasWebChannelSupport()) {
```

Comment:
> @shane-tomlinson @vershwal can we use fx_desktop_v3 or do we need to change it depending on Desktop vs Mobile 

This should be `fx_desktop_v3` if the user is on desktop and `fx_fennec_v1` if on mobile. We have logic to make that selection [here](https://github.com/mozilla/fxa-content-server/blob/f44dcb5829ce657d48cb710424df344cfc5bd021/app/scripts/views/connect_another_device.js#L204:L215). If that snippet could be shared, that'd be a bonus!